starters/rest-api/config/development.yaml

```diff
@@ -74,7 +74,7 @@ mailer:
     # Enable/Disable smtp mailer.
     enable: true
     # SMTP server host. e.x localhost, smtp.gmail.com
-    host: {{ get_env(name="MAILER_HOST", default="localhost") }}
+    host: { { get_env(name="MAILER_HOST", default="localhost") } }
```

Comment:
can you remove the spaces? it will brake the config

Reply:
My bad, the ide reformatted the yaml file, will change it now